tests/integration/mock/CodeActAgent/test_simple_task_rejection/prompt_001.log

```diff
@@ -0,0 +1,335 @@
+
```

Comment:
It's weird that this test is NOT skipped during `regenerate`. `def test_simple_task_rejection()` does have 

```python
@pytest.mark.skipif(
    os.getenv('AGENT') != 'ManagerAgent',
    reason='Currently, only ManagerAgent supports task rejection',
)
```

Anyway, could you please delete this folder?

Reply:
Sure, thanks for your review, I deleted the files. Is this not related to FORCE_USE_LLM=true? IIUC, doesn't the environment variables only play a role in `regenerate_without_llm` but if we run `regenerate_with_llm` in which case $AGENT is irrelevant? This would make sense in the case of using FORCE_USE_LLM=true.

Reply:
Ah there's probably a bug in the script that doesn't correctly set the env variable.